apps/report-documents-not-received/translations/src/en/accessibility.json

```diff
@@ -0,0 +1,33 @@
+{
+    "intro": "This service is run by the Home Office. We want as many people as possible to be able to use this website. For example, that means you should be able to:",
+    "intro-p2-li1": "change colours, contrast levels and fonts",
+    "intro-p2-li2": "zoom in up to 400% without the text spilling off the screen",
+    "intro-p2-li3": "navigate most of the website using just a keyboard",
+    "intro-p2-li4": "navigate most of the website using speech recognition software",
+    "intro-p2-li5": "listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)",
+    "intro-p3-pre-link": "We have also made the website text as simple as possible to understand. ",
+    "intro-p3-link": "AbilityNet",
+    "intro-p3-post-link": " has advice on making your device easier to use if you have a disability.",
+    "reporting-p1": "Reporting accessibility problems with this website",
+    "reporting-p2": "We are always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we're not meeting accessibility requirements, contact us at hof-accessibility@digital.homeoffice.gov.uk.",
+    "reporting-p3": "Read tips on contacting organisations about inaccessible websites.",
+    "enforcement-p1": "Enforcement procedure",
+    "enforcement-p2": "The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’). If you’re not happy with how we respond to your complaint, contact the Equality Advisory and Support Service (EASS).",
+    "enforcement-p3": "If you are in Northern Ireland and are not happy with how we respond to your complaint you can contact the Equalities Commission for Northern Ireland who are responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’) in Northern Ireland.",
+    "non-accessible-header": "Non-accessible content",
+    "compliance-link-p1": "This website is partially compliant with the",
+    "compliance-link-p2": "AA standard, due to the non-compliances listed below.",
+    "non-compliance-aware-issues": "The content listed below is non-accessible for the following reasons.",
+    "non-compliance-1": "On the ‘What reference number can you provide’ page an aria-labelledby or aria-describedby reference exists but the target for the reference does not exist. Screen readers will not pick up on the relationship between the words and the input and so may progress without completing the action. This does not meet the following WCAG standards and guidelines:",
```

Comment:
Im not sure if this wording is common but I think the page title is `Which reference number can you provide` where we have the accessibility issue